<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlButtonsWidget</class>
 <widget class="QWidget" name="ControlButtonsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>51</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>666</width>
    <height>51</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>666</width>
    <height>51</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="0">
        <red>239</red>
        <green>240</green>
        <blue>241</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="0">
        <red>239</red>
        <green>240</green>
        <blue>241</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="0">
        <red>239</red>
        <green>240</green>
        <blue>241</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="0">
        <red>239</red>
        <green>240</green>
        <blue>241</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="backButton">
     <property name="minimumSize">
      <size>
       <width>66</width>
       <height>54</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>66</width>
       <height>54</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#backButton {
	background-color: transparent;
	border-image: url(:assets/prev.png);
	background: none;
	background-repeat: none;
}

#backButton:pressed {
	border-image: url(:assets/prev_p.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="playButton">
     <property name="minimumSize">
      <size>
       <width>66</width>
       <height>54</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>66</width>
       <height>54</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#playButton {
	background-color: transparent;
	border-image: url(:assets/play.png);
	background: none;
	background-repeat: none;
}

#playButton:pressed {
	border-image: url(:assets/play_p.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pauseButton">
     <property name="minimumSize">
      <size>
       <width>66</width>
       <height>54</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>66</width>
       <height>54</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#pauseButton {
	background-color: transparent;
	border-image: url(:assets/pause.png);
	background: none;
	background-repeat: none;
}

#pauseButton:pressed {
	border-image: url(:assets/pause_p.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="stopButton">
     <property name="minimumSize">
      <size>
       <width>66</width>
       <height>54</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>66</width>
       <height>54</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#stopButton {
	background-color: transparent;
	border-image: url(:assets/stop.png);
	background: none;
	background-repeat: none;
}

#stopButton:pressed {
	border-image: url(:assets/stop_p.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="nextButton">
     <property name="minimumSize">
      <size>
       <width>66</width>
       <height>54</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>66</width>
       <height>54</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#nextButton {
	background-color: transparent;
	border-image: url(:assets/next.png);
	background: none;
	background-repeat: none;
}

#nextButton:pressed {
	border-image: url(:assets/next_p.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>18</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="openButton">
     <property name="minimumSize">
      <size>
       <width>66</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>66</width>
       <height>48</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#openButton {
	background-color: transparent;
	border-image: url(:assets/open.png);
	background: none;
	background-repeat: none;
}

#openButton:pressed {
	border-image: url(:assets/open_p.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>21</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="shuffleButton">
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>45</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>141</width>
       <height>45</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
    spacing: 0px;
}

QCheckBox::indicator {
    width: 141px;
    height: 45px;
}

QCheckBox::indicator:unchecked {
    image: url(:assets/shuffle_off.png);
}

QCheckBox::indicator:unchecked:hover {
    image: url(:assets/shuffle_off.png);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(:assets/shuffle_off_p.png);
}

QCheckBox::indicator:checked {
    image: url(:assets/shuffle_on.png);
}

QCheckBox::indicator:checked:hover {
    image: url(:assets/shuffle_on.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(:assets/shuffle_on_p.png);
}

QCheckBox::indicator:indeterminate:hover {
    image:  url(:assets/shuffle_off.png);
}

QCheckBox::indicator:indeterminate:pressed {
    image:  url(:assets/shuffle_off_p.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="repeatButton">
     <property name="minimumSize">
      <size>
       <width>84</width>
       <height>45</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>45</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
    spacing: 0px;
}

QCheckBox::indicator {
    width: 85px;
    height: 45px;
}

QCheckBox::indicator:unchecked {
    image: url(:assets/repeat_off.png);
}

QCheckBox::indicator:unchecked:hover {
    image: url(:assets/repeat_off.png);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(:assets/repeat_off_p.png);
}

QCheckBox::indicator:checked {
    image: url(:assets/repeat_on.png);
}

QCheckBox::indicator:checked:hover {
    image: url(:assets/repeat_on.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(:assets/repeat_on_p.png);
}

QCheckBox::indicator:indeterminate:hover {
    image:  url(:assets/repeat_off.png);
}

QCheckBox::indicator:indeterminate:pressed {
    image:  url(:assets/repeat_off_p.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
